src/tests/testsuite.ts

```diff
@@ -114,49 +117,42 @@ export const createTestSuite: CreateTestsuite = (name, doubleCsrfOptions) => {
           mockResponse,
           csrfToken,
           cookieValue: oldCookieValue,
-        } = generateMocksWithTokenIntenral();
+        } = generateMocksWithTokenInternal();
 
         // reset the mock response to have no cookies (in reality this would just be a new instance of Response)
         mockResponse.setHeader("set-cookie", []);
 
-        const generatedToken = generateToken(mockRequest, mockResponse, {
+        const generatedToken = generateCsrfToken(mockRequest, mockResponse, {
           overwrite: true,
         });
         const newCookieValue = getCookieFromResponse(mockResponse);
-
+        assert.typeOf(oldCookieValue, "string");
+        assert.typeOf(newCookieValue, "string");
+        assert.isNotEmpty(oldCookieValue);
+        assert.isNotEmpty(newCookieValue);
         assert.notEqual(newCookieValue, oldCookieValue);
         assert.notEqual(generatedToken, csrfToken);
       });
 
       it("should throw if csrf cookie is present and invalid, overwrite is false, and validateOnReuse is enabled", () => {
         const { mockRequest, mockResponse, decodedCookieValue } =
-          generateMocksWithTokenIntenral();
-        // modify the cookie to make the token/hash pair invalid
-        signed
-          ? (mockRequest.signedCookies[cookieName] = `s:${sign(
-              (decodedCookieValue as string).split("|")[0] + "|invalid-hash",
-              mockRequest.secret as string,
-            )}`)
-          : (mockRequest.cookies[cookieName] =
-              (decodedCookieValue as string).split("|")[0] + "|invalid-hash");
```

Comment:
Signed option is no longer supported, hmac is inherently signed. Technically the previous hash implementation was also inherently signed. It's redundant, nobody needs this. Thus all tests specific to `signed` option have been removed.